### PR TITLE
Added the volunteers page.

### PR DESCRIPTION
**Pull request by amolkahat**
Hi Sayan I created the page. Please give review on this.

### CODE REVIEW

**Comment by amolkahat** [comment]
@sayanchowdhury please take a look. I fixed volunteers page.

File: css/main.css [hunk 1]
```diff
@@ -5,11 +5,6 @@ body {
 
 a {
   text-decoration: none;
-  cursor: pointer;
-}
-
-.no-text-transform {
-  text-transform: none;
```
Comment (sayanchowdhury) [inline css/main.css:12]: Why remove this?

**Comment by sayanchowdhury** [comment]
The page needs to be based on the base/base_ot.html template. Check the structure of the registration template

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

**Comment by amolkahat** [comment]
@sayanchowdhury Plaease take a look.

**Comment by sayanchowdhury** [comment]
How are you testing the pages?

**Pull request closed without merging**